addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -25,15 +25,20 @@ export class MassMailingHtmlField extends HtmlField {
     }
 
     setup() {
-        super.setup();
-
         useSubEnv({
+            onIframeUpdated: this._updateIframe.bind(this),
             onWysiwygReset: this._resetIframe.bind(this),
         });
+        super.setup();
```

Comment:
Is this really necessary ? Seems like fullwidth version worked fine with the super call at the top

Reply:
Hello @reth-odoo :slightly_smiling_face: 
The issue I had here was that in the super call, the `onIframeUpdated` was retrieved from the env and assigned:
https://github.com/odoo/odoo/blob/saas-17.2/addons/web_editor/static/src/js/backend/html_field.js#L70 
But at that time the `useSubEnv` hasn't been called yet so it was setting it to an empty method instead of using `_updateIframe` :confused: 
I didn't know how to face the issue so I just declared the setup method after the `useSubEnv` :sweat_smile: 

But now that I think about it, it'll maybe work to just overwrite the `this.onIframeUpdated` :thinking:
I'm trying that! :smile:

Reply:
@reth-odoo hi again :slightly_smiling_face: 
Well... Someone broke the display in 17.2 https://tinyurl.com/2cv3tyny 
Investigating why as it broke my fix as well :sweat_smile: (2 vertical scroll bars and one horizontal instead of only the one from the website snippets)
![image](https://github.com/odoo/odoo/assets/107679666/fb9a9557-deb2-467c-8dbc-f96b225cd15d)

Reply:
Just realized it was a different controller for the full-width view, so it's not the same

Indeed here as we don't need to pass the value to children components just overwriting should be fine :crossed_fingers: